### PR TITLE
Fix Java warnings for com.ibm.ws.logging.core

### PR DESCRIPTION
**Pull request by chackoj**
Remove/suppress IDE Java warnings as appropriate.

### CODE REVIEW

**Comment by chackoj** [comment]
#build

**Review by ngmr — APPROVED** [review]
Looks good to me

File: dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingSoftQueue.java [hunk 1]
```diff
@@ -80,30 +80,31 @@ private int getNext(int index) {
      *         are in order from head to tail so it can be iterated normally.
      *
      */
-    @Override
-    public <T> T[] toArray(T[] arr) {
+    @SuppressWarnings("unchecked")
+	@Override
```
Comment (chackoj) [inline dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingSoftQueue.java:84]: bad indenting here

File: dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueue.java [hunk 2]
```diff
@@ -92,15 +92,16 @@ private int getNext(int index) {
...
-    public <T> T[] toArray(T[] arr) {
+    public <X> X[] toArray(X[] arr) {
 
         if (arr == null) {
             return null;
         }
 
-        T[] retMe = (arr.length >= elements.length)
+        @SuppressWarnings("unchecked")
+		X[] retMe = (arr.length >= elements.length)
```
Comment (chackoj) [inline dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueue.java:102]: bad indenting here

File: dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingSoftQueue.java [hunk 3]
```diff
@@ -129,7 +130,8 @@ public boolean isEmpty() {
         return false;
     }
     
-    @Override
+    @SuppressWarnings("unchecked")
+	@Override
```
Comment (chackoj) [inline dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingSoftQueue.java:134]: bad indenting here

**Comment by chackoj** [comment]
#build

**Comment by chackoj** [comment]
#run-libby-bot

**Pull request merged**